src/bika/extras/browser/overrides/setupdata.py

```diff
@@ -596,10 +606,21 @@ def load_artemplate_partitions(self):
                 'Preservation': preservation.Title() if preservation else None,
                 'preservation_uid': preservation.UID()} if preservation else None)
 
+    def get_setup_folder(self, folder_id):
```

Comment:
@Lunga001 I was thinking about making the get_setup_folder a global method but thought against it, What do you think?

Reply:
Please have a look at https://github.com/senaite/senaite.core/blob/2.x/src/senaite/core/exportimport/setupdata/__init__.py#L1263 it seems like the setup is available on the context, so instead of having a function you can have a one liner
`container = self.context.setup.analysiscategories`

Reply:
@Lunga001 I have made the requested changes.